web/wp-config.php

```diff
@@ -4,37 +4,6 @@
  * This file is required in the root directory so WordPress can find it.
  * WP is hardcoded to look in its own directory or one directory up for wp-config.php.
  */
-
-/**
- * Pantheon platform settings. Everything you need should already be set.
- */
-if (file_exists(dirname(__FILE__) . '/wp-config-pantheon.php') && isset($_ENV['PANTHEON_ENVIRONMENT'])) {
-	require_once(dirname(__FILE__) . '/wp-config-pantheon.php');
-} else {
-    require_once dirname(__DIR__) . '/vendor/autoload.php';
-    require_once dirname(__DIR__) . '/config/application.php';
-}
-
-/*
-* Define wp-content directory outside of WordPress core directory
-*/
-define( 'WP_CONTENT_DIR', dirname( __FILE__ ) . '/app' );
-define( 'WP_CONTENT_URL', WP_HOME . '/app' );
-
-/**
- * WordPress Database Table prefix.
- *
- * You can have multiple installations in one database if you give each
- * a unique prefix. Only numbers, letters, and underscores please!
- */
-$table_prefix = getenv( 'DB_PREFIX' ) !== false ? getenv( 'DB_PREFIX' ) : 'wp_';
-
-/* That's all, stop editing! Happy Pressing. */
-
-/** Absolute path to the WordPress directory. */
-if ( ! defined( 'ABSPATH' ) ) {
-	define( 'ABSPATH', dirname( __FILE__ ) . '/' );
-}
-
-/** Sets up WordPress vars and included files. */
-require_once( ABSPATH . 'wp-settings.php' );
\ No newline at end of file
+require_once dirname(__DIR__) . '/vendor/autoload.php';
+require_once dirname(__DIR__) . '/config/application.php';
+require_once ABSPATH . 'wp-settings.php';
```

Comment:
If a user is adding their `wp-config.php` customizations, where should they be added? `config/application.php`? `wp-config.php`? If `wp-config`, we should have a `/* That's all, stop editing! Happy Pressing. */` line so they know where they should put their customizations. I think after the `require_once ABSPATH . `wp-settings.php` may be too late.